website/manual.md

```diff
@@ -1155,13 +1155,13 @@ We are very concerned about making mistakes when adding new APIs. When adding an
 Op to Deno, the counterpart interfaces on other platforms should be researched.
 Please list how this functionality is done in Go, Node, Rust, and Python.
 
-As an example, see how `Deno.rename()` was proposed and added in
+As an example, see how `deno.rename()` was proposed and added in
 [PR #671](https://github.com/denoland/deno/pull/671).
 
 ### Documenting APIs
 
 It is important to document public APIs and we want to do that inline with the
-code. This helps ensure that code and documentation are tightly coupled
+code. This helps ensure that both code and documentation are tightly coupled
```

Comment:
I don't think I agree with this one. Maybe give it a reread?

Reply:
Agreed I'll revert this bit